src/compiler/emitter.ts

```diff
@@ -3090,32 +3090,39 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                 }
             }
 
-            function emitExportMemberAssignments(name: Identifier) {
+            function emitExportMemberAssignmentsInNonSystemModule(name: Identifier) {

```

Comment:
why not just keep this `emitExportMemberAssignments` eith a return at the top and note that it is handled in System emit?